go/vt/vterrors/stack.go

```diff
@@ -0,0 +1,189 @@
+package vterrors
```

Comment:
This is a fork of existing code right? Maybe you can refer to where this is from?